socorro/lib/task_manager.py

```diff
@@ -76,7 +64,6 @@ def __init__(
         idle_delay=7,
         quit_on_empty_queue=False,
         job_source_iterator=default_iterator,
-        heartbeat_func=default_heartbeat,
```

Comment:
Socorro still has this TaskManager and ThreadedTaskManager which is the last remnants for how socorro service processes were structured a long long time ago.

In this PR, I removed the heartbeat function from the processor app as well as the scaffolding for heartbeats for any app that uses the task managers figuring we don't need that anymore. (We can always re-add it if we do need it.)